### DISCUSSION TITLE
Is there a way to conver vector<string> with entry key-vaule pair(ex. ["name:John", "Age:20"]) to json?)

### DISCUSSION THREAD

**Discussion by YunlongB** [nlohmann/json#4404] [Q&A]
Hey There,
I am wondering If there exist a direct way to convert the vector<string> to json, a example of vector<string> is  ["name:John", "Age:20", "Gender: male"] each string contains a key vaule pair that separated by colon.
Appreciate

**Comment by nlohmann** [comment]
No, you would need to parse the input and then process the strings in a second step.
